<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditDriver</class>
 <widget class="QDialog" name="DialogEditDriver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>945</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bitwise</family>
    <pointsize>32</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>858</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/ EDIT DRIVER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="lineTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>858</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>668</x>
     <y>885</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitwise</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>885</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitwise</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxDriverName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>858</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Name</string>
   </property>
   <widget class="QLineEdit" name="lineEditDriverName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>838</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitwise</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
